src/main/java/dk/bec/polonez/reservationsystem/controller/CustomExceptionHandler.java

```diff
@@ -0,0 +1,42 @@
+package dk.bec.polonez.reservationsystem.controller;
+
+import dk.bec.polonez.reservationsystem.exception.AuthenticationException;
+import dk.bec.polonez.reservationsystem.exception.ErrorResponse;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler
+{
```

Comment:
**advice** Minor: Weird code formatting, shouldn't ```{``` be in line 19? (ALT + CTRL + L in IntelliJ)